docs/try_local_system.md

```diff
@@ -442,6 +443,13 @@ You can opt to use your favorite package manager instead.
    cargo install just
    ```
 
+6. Install the `protoc` to compile the proto files. (optional: if gRPC services are enabled)
+   
+   ```shell
+   brew install protoc
+   brew install protobuf
+   ```
```

Comment:
1. I don't think it's optional: we'll need `protoc` to run `cargo check` as well I believe.
2. There's no package called `protoc`, it's only `protobuf`. Keep only the second command.
3. Can you update the instructions for Ubuntu, WSL2 and Windows as well?

Reply:
It would be optional and under a feature flag